pkg/auth/server/login/login.go

```diff
@@ -27,9 +38,17 @@ type LoginFormRenderer interface {
 type LoginForm struct {
 	Action string
 	Error  string
+	Names  LoginFormNames
 	Values LoginFormValues
 }
 
+type LoginFormNames struct {
```

Comment:
you can actually use the same struct for names and values, and call it `LoginFormFields` :)